<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettingsDialog</class>
 <widget class="QDialog" name="CSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>483</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>NitroShare Settings</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QTabWidget" name="Tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="GeneralTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="MachineNameLabel">
         <property name="text">
          <string>Machine name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="MachineName"/>
       </item>
       <item>
        <widget class="QLabel" name="ReceivedFilesLabel">
         <property name="text">
          <string>Directory to store received files in:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="ReceivedFilesBox">
         <item>
          <widget class="QLineEdit" name="ReceivedFilesDirectory"/>
         </item>
         <item>
          <widget class="QPushButton" name="Browse">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="LoadAtStartup">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Load at startup</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CompressFiles">
         <property name="text">
          <string>Compress files (using zlib)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CalculateChecksum">
         <property name="text">
          <string>Calculate checksum of files</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="SystemTray">
         <property name="text">
          <string>System tray only (requires restart)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="Spacer1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="AppearanceTab">
      <attribute name="title">
       <string>Appearance</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Application indicator / system tray icon:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignJustify|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="IconLayout">
         <item row="0" column="1">
          <widget class="QRadioButton" name="Dark">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Dark</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="DarkIcon">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../resource/main.qrc">:/icons/dark.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="LightIcon">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../resource/main.qrc">:/icons/light.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="Light">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Light</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="NotificationsTab">
      <attribute name="title">
       <string>Notifications</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="NotificationsLabel">
         <property name="text">
          <string>Display notifications for the following events:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="NotifyNewPC">
         <property name="text">
          <string>A new PC running NitroShare joins the network</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="NotifyPCQuit">
         <property name="text">
          <string>A PC running NitroShare leaves the network</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="NotifyFilesReceived">
         <property name="text">
          <string>Files are received</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="NotifyError">
         <property name="text">
          <string>An error occurs </string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="Spacer2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="SecurityTab">
      <attribute name="title">
       <string>Security</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QRadioButton" name="AcceptAll">
         <property name="text">
          <string>Automatically accept all files</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="AcceptPattern">
         <property name="text">
          <string>Accept files that match the regular expression below:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="RegExLayout">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QLineEdit" name="AcceptRegEx"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QRadioButton" name="AcceptPrompt">
         <property name="text">
          <string>Always prompt before accepting files</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="AcceptNone">
         <property name="text">
          <string>Do not accept any files</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="Spacer3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="NetworkTab">
      <attribute name="title">
       <string>Network</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="InterfaceLayout">
         <item>
          <widget class="QLabel" name="InterfaceLabel">
           <property name="text">
            <string>Network interface:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="InterfaceList">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="Line1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Warning">
         <property name="text">
          <string>&lt;b&gt;Warning:&lt;/b&gt; changing the settings below may cause NitroShare to stop functioning or use excessive bandwidth. Proceed with caution.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="Line2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="BroadcastBox">
         <property name="title">
          <string>Broadcast Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="BroadcastPortLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Broadcast port:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="BroadcastPort">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maxLength">
             <number>32767</number>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="BroadcastIntervalLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Broadcast interval (in milliseconds):</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="BroadcastInterval">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="TimeoutIntervalLabel">
            <property name="text">
             <string>Timeout interval (in seconds):</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="TimeoutInterval">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="TransmissionBox">
         <property name="title">
          <string>Transmission Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="TransmissionPortLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>TCP port:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="TransmissionPort">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="Spacer4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>CSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>CSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Browse</sender>
   <signal>clicked()</signal>
   <receiver>CSettingsDialog</receiver>
   <slot>OnBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnBrowse()</slot>
 </slots>
</ui>
